components/application-mgt/org.wso2.carbon.identity.application.common/pom.xml

```diff
@@ -65,6 +65,7 @@
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
                         <Import-Package>
                             javax.cache,
+                            javax.cache.event.*,
```

Comment:
Do we need * here ? the import in previous line doesn't have .* and still there are 3 classes used under it. I think we can remove the * here. Please check.